ui/nim-status-client.pro

```diff
@@ -23,6 +23,7 @@ TRANSLATIONS += \
 
 OTHER_FILES += $$files("$$PWD/*.qml", true)
 OTHER_FILES += $$files("$$PWD/*.js", true)
+OTHER_FILES += $$files("$$PWD/../src/*.nim", true)
```

Comment:
Searching files in qt creator (ctrl+k) is already very slow (at least for me), with all `*.nim` files it will probably be even slower :disappointed: But I understand it's your IDE of choice for nim development and can't forbid it here :man_shrugging:

Reply:
What's your QtC version? Here in 8.0.1 it's super fast

Reply:
It's unclear why this change is needed for this commit

Reply:
Was 7.0.x, just have updated to 8.0.1, it's super fast! :fireworks:

Reply:
Yeah it is, also the cmake/C++ support is so much better